fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -617,14 +617,21 @@ columnNameWithComment
 // ------------------------------------------- Task Statement ----------------------------------------------------------
 
 submitTaskStatement
-    : SUBMIT TASK qualifiedName? properties?
+    : SUBMIT TASK qualifiedName?
+        ( properties?
+          | taskScheduleDesc?
```

Comment:
task schedule info should be put before properties.

Reply:
done. right now it can be in arbitrary order